commit 3e3050a313f15a7239cae914c2823a99b573cd38 (refs/changes/83/1683/3)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2017-03-18T00:07:47+00:00 (2 years, 7 months ago)
    
    CNS-196 CNS dumps core on iquery
    Reviewed by: Brittany Wald <brittany.wald@joyent.com>

diff --git a/package.json b/package.json
index ea1f6ed..602545a 100644
--- a/package.json
+++ b/package.json
@@ -34,7 +34,7 @@
     "ldapjs": "1.0.0",
     "lru-cache": "2.7.0",
     "mooremachine": "2.1.0",
-    "mname": "1.3.4",
+    "mname": "1.3.5",
     "redis": "2.1.0",
     "restify": "4.0.3",
     "restify-clients": "1.1.1",
